--------------------Moonraker Log Start--------------------
data_path: /home/pi/printer_data
is_default_data_path: False
config_file: /home/pi/klipper_config/moonraker.conf
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
log_file: /home/pi/klipper_logs/moonraker.log
software_version: v0.7.1-780-gdde9bcc
python_version: 3.7.3 (default, Jan 22 2021, 20:04:44)  [GCC 8.3.0]

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125

[file_manager]

[data_store]
temperature_store_size = 600
gcode_store_size = 1000

[authorization]
force_logins = True
cors_domains = 
	*.local
	*.lan
	*://app.fluidd.xyz
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128

[octoprint_compat]

[history]

[update_manager]
enable_auto_refresh = True

[update_manager client fluidd]
type = web
repo = cadriel/fluidd
path = ~/fluidd

#################################################################
All Configuration Files:
/home/pi/klipper_config/moonraker.conf
#################################################################
Unsafe Shutdown Count: 92

System Info:

***python***
  version: sys.version_info(major=3, minor=7, micro=3, releaselevel='final', serial=0)
  version_string: 3.7.3 (default, Jan 22 2021, 20:04:44)  [GCC 8.3.0]

***cpu_info***
  cpu_count: 4
  bits: 32bit
  processor: armv7l
  cpu_desc: ARMv7 Processor rev 4 (v7l)
  serial_number: db714f34
  hardware_desc: BCM2835
  model: Raspberry Pi 3 Model B Plus Rev 1.3
  total_memory: 892584
  memory_units: kB

***sd_info***
  manufacturer_id: 03
  manufacturer: Sandisk
  oem_id: 5344
  product_name: SK32G
  product_revision: 8.0
  serial_number: bf3b0342
  manufacturer_date: 2/2021
  capacity: 29.7 GiB
  total_bytes: 31914983424

***distribution***
  name: Raspbian GNU/Linux 10 (buster)
  id: raspbian
  version: 10
  version_parts: {'major': '10', 'minor': '', 'build_number': ''}
  like: debian
  codename: buster
  release_info: {'name': '0.', 'version_id': '4.0', 'id': 'fluiddpi'}

***virtualization***
  virt_type: none
  virt_identifier: none

***network***

***canbus***

Systemd unit moonraker.service:
unit_name: moonraker.service
is_default: True
Properties:
**Requires=['-.mount', 'sysinit.target', 'system.slice', 'network-online.target']
**After=['-.mount', 'sysinit.target', 'system.slice', 'network-online.target', 'systemd-journald.socket', 'basic.target']
**SupplementaryGroups=['moonraker-admin']
**EnvironmentFiles=/home/pi/printer_data/systemd/moonraker.env
**ExecStart=/home/pi/moonraker-env/bin/python $MOONRAKER_ARGS
**WorkingDirectory=/home/pi/moonraker
**FragmentPath=/etc/systemd/system/moonraker.service
**Description=API Server for Klipper SV1
**User=pi

Systemd unit klipper.service:
unit_name: klipper.service
is_default: True
Properties:
**Description=Starts klipper on startup
**ExecStart=/home/pi/klippy-env/bin/python /home/pi/klipper/klippy/klippy.py /home/pi/klipper_config/printer.cfg -l /home/pi/klipper_logs/klippy.log -a /tmp/klippy_uds
**FragmentPath=/etc/systemd/system/klipper.service
Klipper Version: v0.10.0-588-g51da02b7
2023-02-19 20:07:15,373 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.0.166
2023-02-19 20:09:20,369 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.166
2023-02-19 20:09:30,710 [app.py:log_request()] - 201 POST /server/files/upload (192.168.0.166) [_TRUSTED_USER_] 18.71ms
2023-02-19 20:09:48,669 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2023-02-19 20:09:48,693 [websockets.py:build_error()] - JSON-RPC Request Error: 503
Klippy Host not connected
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 253, in execute_method
    result = await callback(params)
  File "/home/pi/moonraker/moonraker/websockets.py", line 351, in func
    user=sc.user_info))
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 471, in request
    raise ServerError("Klippy Host not connected", 503)
utils.ServerError: Klippy Host not connected
2023-02-19 20:09:48,924 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2023-02-19 20:09:49,260 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2023-02-19 20:09:49,264 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2023-02-19 20:09:49,511 [job_state.py:_handle_started()] - Job state initialized: standby
2023-02-19 20:09:49,789 [klippy_connection.py:_check_ready()] - Klippy ready
2023-02-19 20:09:50,266 [data_store.py:_init_sensors()] - Configuring available sensors: ['heater_bed', 'temperature_sensor raspberry_pi', 'temperature_sensor mcu_temp', 'extruder']
